src/components/index.js

```diff
@@ -24,5 +24,6 @@ require('./scene/embedded');
 require('./scene/inspector');
 require('./scene/fog');
 require('./scene/keyboard-shortcuts');
+require('./scene/post-processing');
```

Comment:
`postprocessing` is clearest to me, i wouldn't know `deferred` unless i had the context. just wanted to rm the hyphen so you can do `sceneEl.systems.postprocessing` :)